### PR TITLE
Update parity-bridges-common

### PR DESCRIPTION
**Pull request by fewensa**
update pangolin-pangoro dependencies
related:
https://github.com/darwinia-network/darwinia-common/pull/808
https://github.com/darwinia-network/darwinia-common/pull/811

### CODE REVIEW

File: task/task-pangolin-pangoro/src/config.rs [hunk 1]
```diff
@@ -146,6 +150,7 @@ impl ChainInfoConfig {
             port: host_port.2,
             signer: except_signer.or_else(|| self.signer.clone()),
             signer_password: self.signer_password.clone(),
+            transactions_mortality: Some(4),
```
Comment (AurevoirXavier) [inline task/task-pangolin-pangoro/src/config.rs:153]: Why 4 here?

Comment (HackFisher) [reply task/task-pangolin-pangoro/src/config.rs:153]: 4 might be too short, only 4 block time morality for extrinsics(24 s), and better to equal or less than 256, cause `BlockHashCountForCrab` is 256

https://github.com/darwinia-network/darwinia/blob/a965cd8d2c48e98644b0edbffe846f31cbfb3f6f/runtime/common/src/lib.rs#L67

Comment (AurevoirXavier) [reply task/task-pangolin-pangoro/src/config.rs:153]: @WoeOm how polkadot js calculate this arg

Comment (fewensa) [reply task/task-pangolin-pangoro/src/config.rs:153]: There is generate config template, we can put this document to template file. and there will be change to 256.

**Review by HackFisher — APPROVED** [review]

File: task/task-pangolin-pangoro/src/config.rs (hunk 1 above)
Comment (AurevoirXavier) [reply task/task-pangolin-pangoro/src/config.rs:153]: Set default to 256 would be better.

**Pull request merged**